api/server/server.go

```diff
@@ -430,9 +423,6 @@ func WithAgentFromEnv() Option {
 			if runnerURL == "" {
 				return errors.New("no FN_RUNNER_API_URL provided for an Fn NuLB node")
 			}
-			if s.datastore != nil {
-				return errors.New("lb nodes must not be configured with a datastore (FN_DB_URL)")
-			}
```

Comment:
i removed a few of these, this is being validated elsewhere in a 'more consistent way' still - see right about here: https://github.com/fnproject/fn/pull/1471/files#diff-44a527ea72ea9572e1cf71c7f1e2a0f0L568